scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -167,13 +166,13 @@ def _test_splined_vanadium_exists(instrument, run_details):
                          " \nHave you run the method to create a Vanadium spline with these settings yet?\n")
 
 
-def _crop_spline_to_percent_of_max(spline, input_ws):
+def _crop_spline_to_percent_of_max(spline, input_ws, spectrum):
```

Comment:
If `spectrum` is only being used as the name of the `OutputWorkspace`  can we instead change this variable name to be `output_ws` or similar. I think the current implementation makes the code less readable and not obvious that `spectrum` could be either a string or a workspace type and both are valid.

Reply:
Ok, I've changed spectrum to output_workspace